pkg/storage/common.go

```diff
@@ -0,0 +1,98 @@
+package storage
```

Comment:
Remove this change from the PR if it is unrelated to the issue we are resolving.

Reply:
removed.